app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/MultipleDisposalsTriageController.scala

```diff
@@ -62,25 +67,55 @@ class MultipleDisposalsTriageController @Inject() (
   def guidanceSubmit(): Action[AnyContent] = authenticatedActionWithSessionData.async { implicit request =>
     withMultipleDisposalTriageAnswers(request) {
       case (_, _, _) =>
-        Ok("not implemented yet")
+        Redirect(routes.MultipleDisposalsTriageController.howManyDisposals())
+    }
+  }
+
+  def howManyDisposals(): Action[AnyContent] = authenticatedActionWithSessionData.async { implicit request =>
+    withMultipleDisposalTriageAnswers(request) {
+      case (_, _, answers) =>
+        val numberOfDisposals = answers.fold(_.numberOfProperties, c => Some(c.numberOfProperties))
+        val form              = numberOfDisposals.fold(numberOfPropertiesForm)(numberOfPropertiesForm.fill)
+        Ok(howManyProperties(form))
+    }
+  }
+
+  def howManyDisposalsSubmit(): Action[AnyContent] = authenticatedActionWithSessionData.async { implicit request =>
+    withMultipleDisposalTriageAnswers(request) {
+      case (_, _, _) =>
+        numberOfPropertiesForm
+          .bindFromRequest()
+          .fold(
+            formWithErrors =>
+              BadRequest(
+                howManyProperties(formWithErrors)
+              ), { numberOfProperties =>
+              if (numberOfProperties > 1)
```

Comment:
here you need to update the session data with the new answer - if the journey is a `FillingOutReturn` you need to update the draft return as well. Otherwise, if it's a `StartingNewDraftReturn` you need to make the new draft return from multile to single if the answer is one.